R/FDAClassifTask.R

```diff
@@ -0,0 +1,45 @@
+#' @title Functional data classification task.
+#'
+#' @description
+#' Create a functional data classification task. This means that some features
+#' in the task will be so-called functional covariates / functional features,
+#' measured on a grid or time scale.
```

Comment:
Yes, it think  "channel" is better, but bernd don't like this . more explaination is added.